File: backend/components/metiers/administration_publique/utils/js/samples/sample_js_helper.test.js
// Test du helper JS d’exemple (clé en main)
const { sampleJsHelper } = require('./sample_js_helper');
describe('sampleJsHelper', () => {
  it('should log and return true', () => {
    expect(sampleJsHelper('clé en main')).toBe(true);
  });
});

File: backend/components/metiers/administration_publique/utils/js/samples/sample_js_helper.test.py
# Test du helper JS d’exemple (Python, clé en main)
from .sample_js_helper import sample_js_helper
def test_sample_js_helper(capsys):
    assert sample_js_helper('clé en main') is True
    captured = capsys.readouterr()
    assert '[JS SAMPLE] Exemple:' in captured.out
